salt/states/beacon.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+'''
+Management of the Salt beacons
+==============================================
+
+.. code-block:: yaml
+
+    ps:
+      beacon.present:
+        - enable: False
+        - salt-master: running
+        - apache2: stopped
+
+    sh:
+      beacon.present:
+
+    load:
+      beacon.present:
+        - 1m:
+            - 0.0
+            - 2.0
+        - 5m:
+            - 0.0
+            - 1.5
+        - 15m:
+            - 0.1
+            - 1.0
+
+'''
+
```

Comment:
If you add a `from __future__ import absolute_import` right here, that will clean up the pylint error.